test-frame-log-collector/src/test/java/io/skodjob/testframe/LogCollectorIT.java

```diff
@@ -258,7 +257,7 @@ void testCollectFromNonExistentNamespace() {
     void testChangingCustomResources() throws IOException {
         LogCollector localLogCollector = new LogCollectorBuilder()
             .withRootFolderPath(getFolderPathForTest())
-            .withResources(SECRET)
+            .withNamespacedResources(SECRET)
```

Comment:
Here can be also the `withKubeCmdClient()` and `withKubeClient()` methods instead of setters few lines below.